content/developer/reference/user_interface/views/include/list.rst

```diff
@@ -0,0 +1,569 @@
+
+:edit:
+  boolean_
+
+  Disable/enable record editing on the view.
+
+  If the ``edit`` attribute is set to ``false``, the ``editable`` option will be ignored.
+
+:delete:
+  boolean_
+
+  Disable/enable record deletion on the view through the **Action** dropdown.
+
+:import:
+  boolean_
+
+  Disable/enable record import data on the view.
+
+:export_xlsx:
+  boolean_
+
+  Disable/enable record export data on the view.
+
+:editable:
+  string_ chooses from ``top`` or ``bottom``
+
+  by default, selecting a list view's row opens the corresponding
+  :ref:`form view <reference/user_interface/views/form>`. The ``editable`` attributes
+  makes the list view itself editable in-place.
+
+  Valid values are ``top`` and ``bottom``, making *new* records appear respectively at
+  the top or bottom of the list.
+
+  The architecture for the inline :ref:`form view <reference/user_interface/views/form>`
+  is derived from the list view. Most attributes valid on a :ref:`form view
+  <reference/user_interface/views/form>`'s fields and buttons are thus accepted by list
+  views although they may not have any meaning if the list view is non-editable.
+
+  If the ``edit`` attribute is set to ``false``, the ``editable`` option **will be ignored**.
+
+:multi_edit:
+  ``1``
+
+  editable or not editable list can activate the multi-editing feature by defining the
+  `multi_edit="1"`
+
+:default_group_by:
+  string_ :ref:`model <reference/orm/model>` field name
+
+
+  whether the list view should be grouped if no grouping is specified via the action or
+  the current search. Should be the name of the field to group by when no grouping is
+  otherwise specified
+
+:default_order:
+  `Comma-separated values`_
+
+  overrides the ordering of the view, replacing the model's order (:attr:`-odoo.models.BaseModel._order` model attribute).
+  The value is a comma-separated list of :ref:`fields <reference/orm/fields>`, postfixed by ``desc`` to
+  sort in reverse order:
+
+  .. code-block:: xml
+
+    <tree default_order="sequence,name desc">
+      ...
+    </tree>
+
+:decoration-{$name}:
+  :ref:`python expression <user_interface/views/python_expression>` that defines a boolean_
+
+  allow changing the style of a cell's text based on the corresponding
+  record's attributes.
+
+  ``{$name}`` can be ``bf`` (``font-weight: bold``), ``it``
+  (``font-style: italic``), or any `bootstrap contextual color`_ (``danger``,
+  ``info``, ``muted``, ``primary``, ``success`` or ``warning``).
+
+  Define a conditional display of a record in the style of a row's text based
+  on the corresponding record's attributes. For each record, the expression is
+  evaluated with the record's attributes as context values and if ``true``, the
+  corresponding style is applied to the row.
+
+  .. code-block:: xml
+
+    <tree decoration-danger="field_qty > field_limit">
+      ...
+    </tree>
+
+:limit:
+  integer_
+
+  the default size of a page. It must be a positive integer
```

Comment:
add the default: 
80 for list view, 
40 for x2many list in form view (to be verified)